### ISSUE TITLE
precision stones titles

### ISSUE THREAD

**Issue by JoKillerPT** [marcelo-mason/PreciousStones#1124]
the plugin have support to titles on farewell and welcome messages

**Comment by eUipKh** [comment]
No, you can disable the flags **welcome-message**, **farewell-message** and use
https://github.com/marcelo-mason/PreciousStones/wiki/Flag-Reference#command-on-enter
https://github.com/marcelo-mason/PreciousStones/wiki/Flag-Reference#command-on-exit
